docs/dispatcher/errors.rst

```diff
@@ -15,13 +15,13 @@ If you specify errors handler for dispatcher - it will be used for all handlers
 
 .. code-block:: python
 
-    @router.error(ExceptionTypeFilter(MyCustomException), F.update.message.as_("message"))
+    @router.errors(ExceptionTypeFilter(MyCustomException), F.update.message.as_("message"))
```

Comment:
error**s** is planned to be reomoved (use error instead), all other events are indicated in the singular

Reply:
Wow... Isn't it a breaking change?

Reply:
done